### PR TITLE
AIDA-665

### PR DESCRIPTION
**Pull request by jakevb**
Check README for clarity and accuracy explaining the different types of ontology.  Also updated latest versions of SeedlingOntology, SeedlingOwlOntology, and LDCOwlOntology.

### CODE REVIEW

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# AIF Ontologies
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
# AIDA Ontologies
```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+# Differences Between Ontology and OwlOntology
+
+Some ontologies have a corresponding file with a similar name such as SeedlingOntology and SeedlingOwlOntology. In these
+cases, the Owl ontology signifyies that the rdf file is compatible with strict owl, in particular using the correct syntax
+for more complex properties of classes such as RangeIncludes. The non-Owl ontology uses a simpler structure for easier extraction
+of important values.  Both the Owl and non-Owl ontology should define the same vocabulary.
+
+## AidaDomainOntologiesCommon
+
+A collection of top level classes most other ontologies reference as the highest level class.  Additionally, there are
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
A collection of top level classes all Domain Ontologies reference.  Additionally, there are
```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+also a few universal classes that include important properties such as the CanHaveNumericValue class.  This ontology was
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
also a few universal classes that include important properties such as the CanHaveNumericValue class.  This ontology is a hand-written source file and is not generated.
```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+created by individually selecting those class values and not from any other source of data.
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion

```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+## EntityOntology
+
+A collection of Entity classes determined by the Ontology working group.  Entity referring to
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
A collection of Entity classes determined by the AIDA Ontology Working Group. The term 'Entity' refers to
```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+a physical being or intangible concept that either interacts or is subject to interaction.  This in combination with Events
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
a physical being or intangible concept that either interacts or is subject to interaction.  This, in combination with Events
```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+and Relations ontologies contain the vocabulary that will be used for defining messages post-M18 evaluation.
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
and Relations ontologies, contains the vocabulary that will be used for defining messages post-M18 evaluation.
```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+
+## EventOntology
+
+A collection of Event classes determined by the Ontology working group.  Event referring to
+an experience that occurs when two or more Events interact in some way.  Some examples being building weaponry, signing a
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
something that occurs when two or more Entities interact in some way.  Some examples being building weaponry, signing a
```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+treaty, and the moving of people from one place to another. This in combination with Entity
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
treaty, and the moving of people from one place to another. This, in combination with Entity
```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+and Relations ontologies contain the vocabulary that will be used for defining messages post-M18 evaluation.
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
and Relations ontologies, contains the vocabulary that will be used for defining messages post-M18 evaluation.
```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+
+## InterchangeOntology
+
+The main purpose of this ontology is to contain the Interchange vocabulary and to be used by performers using strict OWL
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
This ontology contains the Interchange vocabulary and is written in strict OWL. 
```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+as reference to the classes and properties that are used in our not-strict OWL ontologies. In particular, it contains
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
In particular, it contains
```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 13]
```diff
@@ -0,0 +1,66 @@
...
+the OWL definitions for domain and range which are usually omitted from the not-strict OWL ontologies.
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
the OWL definitions for domain and range which describe how each property interacts with defined classes.
```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 14]
```diff
@@ -0,0 +1,66 @@
...
+
+# LDCOntology
+
+The ontology was given by LDC as an evolution of the SeedlingOntology and will be used for the M18 evaluation.  This will eventually
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
This Domain Ontology was given by LDC as an evolution of the SeedlingOntology and will be used for the M18 evaluation. 
```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 15]
```diff
@@ -0,0 +1,66 @@
...
+replace SeedlingOntology once the original LDC data has been removed from our programs and updated with the newest version.
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion

```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 16]
```diff
@@ -0,0 +1,66 @@
...
+
+## LDCOwlOntology
+
+The ontology was given by LDC as an evolution of the SeedlingOwlOntology and will be used for the M18 evaluation.  This will eventually
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
This Domain Ontology was given by LDC as an evolution of the SeedlingOwlOntology and will be used for the M18 evaluation.
```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 17]
```diff
@@ -0,0 +1,66 @@
...
+replace SeedlingOwlOntology once the original LDC data has been removed from our programs and updated with the newest version.
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion

```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 18]
```diff
@@ -0,0 +1,66 @@
...
+
+## EntityOntology
+
+A collection of Entity classes determined by the Ontology working group.  Entity referring to
+a physical being or intangible concept that either interacts or is subject to interaction.  This in combination with Events
+and Relations ontologies contain the vocabulary that will be used for defining messages post-M18 evaluation.
+
+## EventOntology
+
+A collection of Event classes determined by the Ontology working group.  Event referring to
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
A collection of Event classes determined by the Ontology working group.  The term 'Event' refers to
```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 19]
```diff
@@ -0,0 +1,66 @@
...
+replace SeedlingOntology once the original LDC data has been removed from our programs and updated with the newest version.
+
+## LDCOwlOntology
+
+The ontology was given by LDC as an evolution of the SeedlingOwlOntology and will be used for the M18 evaluation.  This will eventually
+replace SeedlingOwlOntology once the original LDC data has been removed from our programs and updated with the newest version.
+
+## RelationOntology
+
+A collection of Relation Type classes determined by the Ontology working group.  Relations referring to
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
A collection of Relation Type classes determined by the Ontology Working Group. The term 'Relation' refers to
```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 20]
```diff
@@ -0,0 +1,66 @@
...
+A specification of the connection between two or more Entities. This includes such things as the material used to create
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
a connection between two or more Entities. This includes such things as the material used to create
```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 21]
```diff
@@ -0,0 +1,66 @@
...
+a weapon, the nationality of an individual and the title an employee holds. This in combination with Entities
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
a weapon, the nationality of an individual and the title an employee holds. This, in combination with Entities
```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 22]
```diff
@@ -0,0 +1,66 @@
...
+and Events ontologies contain the vocabulary that will be used for defining messages post-M18 evaluation.
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
and Events ontologies, contains the vocabulary that will be used for defining messages post-M18 evaluation.
```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 23]
```diff
@@ -0,0 +1,66 @@
...
+
+## SeedlingOntology
+
+The ontology created from the first cut of data from LDC and was primarily used by performers during the M9 evaluation.
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
This Domain Ontology was created from the first cut of data from LDC and was primarily used by performers during the M9 evaluation.
```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 24]
```diff
@@ -0,0 +1,66 @@
...
+Ideally, this will become a legacy ontology replaced by LDCOntology, but until we completely remove data from pre-M9, this
+Ontology file will remain for performers to use.
+
+## SeedlingOwlOntology
+
+The ontology created from the first cut of data from LDC and was primarily used by performers during the M9 evaluation.
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
This Domain Ontology was created from the first cut of data from LDC and was primarily used by performers during the M9 evaluation.
```

File: src/main/resources/com/ncc/aif/ontologies/README.md (hunk 7 above)
Comment (craigwarsaw) [inline src/main/resources/com/ncc/aif/ontologies/README.md]: With the new organization that I recommended in the top-level comment for this PR, you can remove the starting with sentence "This in combination with" and replace it with a similar sentence in the main "Program-Wide Ontology Files" section. Like "The Entities, Events, and Relations ontologies contain..."

File: src/main/resources/com/ncc/aif/ontologies/README.md (hunk 8 above)
Comment (craigwarsaw) [inline src/main/resources/com/ncc/aif/ontologies/README.md]: I think you mean "Entities interact in some way", not "Events interact".

File: src/main/resources/com/ncc/aif/ontologies/README.md (hunk 11 above)
Comment (craigwarsaw) [inline src/main/resources/com/ncc/aif/ontologies/README.md]: I think "The main purpose of this ontology is to contain the Interchange vocabulary" is correct, but I'm not sure that the rest is correct.  Let's discuss to help me understand this statement, and the following sentence starting with "In particular".

File: src/main/resources/com/ncc/aif/ontologies/README.md (hunk 15 above)
Comment (craigwarsaw) [inline src/main/resources/com/ncc/aif/ontologies/README.md]: "once the original seedling data"

File: src/main/resources/com/ncc/aif/ontologies/README.md (hunk 14 above)
Comment (craigwarsaw) [inline src/main/resources/com/ncc/aif/ontologies/README.md]: In the new structure that I recommended in the top-level comment of this PR, put this sentence in the main section".
Also change "given" to "provided"

**Comment by jakevb** [comment]
Craig, I have incorporated almost all of the changes both you and Eddie have suggested.  The only one that might still need some clarification is your question regarding the InterchangeOntology.  I have updated that section to improve clarity and incorporate Eddie's suggestions, but if you still have questions, we can discuss what might need to be changed.

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 25]
```diff
@@ -0,0 +1,62 @@
+# AIDA Ontologies
+
+The following is a listing of the ontologies found at AIDA-Interchange-Format/src/main/resources/com/ncc/aif/ontologies.
+Each ontology will contain a short description of what they are built from, what they are used for,
```
Comment (craigwarsaw) [inline src/main/resources/com/ncc/aif/ontologies/README.md]: I think you meant "Each section in this README contains..."

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 26]
```diff
@@ -0,0 +1,62 @@
...
+This Domain Ontology created from the first cut of data from LDC and was primarily used by performers during the M9 evaluation.
+Ideally, this will become a legacy ontology replaced by LDCOntology and LDCOwlOntology, but until we completely remove data from pre-M9, these
+Ontology files will remain for performers to use.
+
+- SeedlingOntology
+- SeedlingOwlOntology
+
+# Program-Wide Ontology Files
+
+The Entities, Events, and Relations ontologies contain the vocabulary that will be used for definning messages post-M18 evaluation.
```
Comment (craigwarsaw) [inline src/main/resources/com/ncc/aif/ontologies/README.md]: MIsspelled "defining"

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 27]
```diff
@@ -0,0 +1,62 @@
...
+## EventOntology
+
+A collection of Event classes determined by the AIDA Ontology Working Group. The term 'Event' refers to
+something that occurs when two or more Entities interact in some way.  Some examples being building weaponry, signing a
+treaty, and the moving of people from one place to another.
+
+## RelationOntology
+
+A collection of Relation Type classes determined by the AIDA Ontology Working Group. The term 'Relation' refers to
+A connection between two or more Entities. This includes such things as the material used to create
```
Comment (craigwarsaw) [inline src/main/resources/com/ncc/aif/ontologies/README.md]: The first letter of this line "A" should be "a"

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 28]
```diff
@@ -0,0 +1,62 @@
+# AIDA Ontologies
+
+The following is a listing of the ontologies found at AIDA-Interchange-Format/src/main/resources/com/ncc/aif/ontologies.
+Each section in this README contains a short description of what they are built from, what they are used for,
+and any other points of relevant information that might be of some use to individuals using these files.
+
+## Differences Between *Ontology and *OwlOntology Files
+
+Some ontologies have a corresponding file with a similar name such as SeedlingOntology and SeedlingOwlOntology. In these
+cases, the *OwlOntology files signifies that the rdf file is compatible with strict OWL, in particular using the correct syntax
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
cases, the *OwlOntology file uses strict OWL and won't include third-party properties/classes 
```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 29]
```diff
@@ -0,0 +1,62 @@
...
+in lieu of *Ontology file properties like schema:rangeIncludes. The *Ontology files are not restricted and can make use of third-party properties to simplify definitions.
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
like schema:rangeIncludes. The *Ontology files are not restricted and make use of third-party properties/classes to simplify definitions.
```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 30]
```diff
@@ -0,0 +1,62 @@
...
+Both the *OwlOntology and *Ontology files define the same vocabulary.
+
+# Common Ontology Files
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
# AIF Ontology Files
```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 31]
```diff
@@ -0,0 +1,62 @@
...
+# LDC Ontology Files
+
+This Domain Ontology was provided by LDC as an evolution of the SeedlingOntology and SeedlingOwlOntology and will be used for the M18 evaluation.
+
+- LDCOntology
+- LDCOwlOntology
+
+# Seedling Ontology Files
+
+This Domain Ontology created from the first cut of data from LDC and was primarily used by performers during the M9 evaluation.
```
Comment (ecurley-nextcentury) [inline src/main/resources/com/ncc/aif/ontologies/README.md]:
```suggestion
This Domain Ontology was created from the first cut of data from LDC and was primarily used by performers during the M9 evaluation.
```

File: src/main/resources/com/ncc/aif/ontologies/README.md [hunk 32]
```diff
@@ -0,0 +1,64 @@
...
+
+### AidaDomainOntologiesCommon
+
+A collection of top level classes all Domain Ontologies reference.  Additionally, there are
+also a few universal classes that include important properties such as the CanHaveNumericValue class.  This ontology is
+a hand-written source file and is not generated.
+
+### InterchangeOntology
+
+This ontology contains the Interchange vocabulary and is written in strict OWL. In particular, it contains
```
Comment (craigwarsaw) [inline src/main/resources/com/ncc/aif/ontologies/README.md]: After discussion with @jakevb, I think I understand where I was confused.  I recommend changing the entire sentence starting from "In particular..." to the following.

It defines the structure of the vocabulary, including the hierarchy of terms (via the "domain" schema definition) and the possible datatypes of terms (via the "range" schema definition).

@ecurley-nextcentury - do you agree with the rewording of this?

Comment (ecurley-nextcentury) [reply src/main/resources/com/ncc/aif/ontologies/README.md]: All RDF is represented as triples (**subject**, **predicate**, **object**). "Domain" refers to what types a particular **predicate** can have as **subject** and "range" refers to what types that **predicate** can have as **object**. If these terms are confusing things, we should just remove them. We can probably change the description to something like, "This ontology defines the Interchange vocabulary and how the defined classes and properties may interact". I'll leave it to you and Jake to come up with something that works. Let me know if you have any questions.

Comment (craigwarsaw) [reply src/main/resources/com/ncc/aif/ontologies/README.md]: Your recommendation is good with me.

Comment (jakevb) [reply src/main/resources/com/ncc/aif/ontologies/README.md]: Sounds good.  I incorporated Eddie's suggested change.

**Pull request merged**